src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.Arrays;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
```

Comment:
Don't begin class or method implementation with an empty line. Consider removing the blank line at the start of the class definition.